.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,75 @@
+name: publish-pypi
```

Comment:
The OIDC publishing to PYPI is scoped the workflow file. I went ahead and added this file to PYPI but we should make a note / comment here to not rename the files!

Reply:
Gotcha, thanks for adding it. Will add a comment